lib/project_types/rails/commands/create.rb

```diff
@@ -29,7 +29,7 @@ def call(args, _name)
 
         ruby_version = Ruby.version(@ctx)
         @ctx.abort(@ctx.message('rails.create.error.invalid_ruby_version')) unless
-          ruby_version.satisfies?('~>2.5') || ruby_version.satisfies?('<3.1')
+          ruby_version.satisfies?('~>2.5') && ruby_version.satisfies?('<3.1')
```

Comment:
This was something I just added and I'm seeing now that it's wrong, but the proposed fix would fail for 3.0.

We could do this:

`ruby_version.satisfies?('~>2.5') || (ruby_version.satisfies?('>=3') && ruby_version.satisfies?('<3.1'))

or...

`ruby_version.satisfies?('~>2.5') || ruby_version.satisfies?('~>3.0.0')`